### ISSUE TITLE
[Dependency] Detected engine 11 but could not load: libstdc++.so.6

### ISSUE THREAD

**Issue by Anon-Exploiter** [Anon-Exploiter/csgo-server#4]
The file `/home/umar_0x01/csgo-ds/csgo/addons/metamod/bin/metamod-fatal.log` contained the following error which doesn't let **sourcemod** or **metamod** to run! 

This doesn't really show up anywhere in the srcds_run logs. 

Log contains:
```bash
└─$ cat metamod-fatal.log
L 06/22/2021 - 18:15:45: Detected engine 11 but could not load: libstdc++.so.6: cannot open shared object file: No such file or directory
L 06/22/2021 - 18:25:00: Detected engine 11 but could not load: libstdc++.so.6: cannot open shared object file: No such file or directory
```

The following fixes it: 
```bash
sudo apt-get install lib32stdc++6
```

**Comment by SpawnTerror** [comment]
Doesn't work. 


# sudo apt-get install lib32stdc++6

Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
lib32stdc++6 is already the newest version (11.1.0-1ubuntu1~21.04).

The following packages were automatically installed and are no longer required:
  libdeflate0 libgd3 libjbig0 libjpeg-turbo8 libjpeg8 libnginx-mod-http-geoip2
  libnginx-mod-http-image-filter libnginx-mod-http-xslt-filter libnginx-mod-mail libnginx-mod-stream
  libnginx-mod-stream-geoip2 libtiff5 libwebp6 nginx-common nginx-core
Use 'sudo apt autoremove' to remove them.
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.


So it is installed.
metamod-fatal.log
L 01/20/2022 - 12:07:12: Detected engine 11 but could not load: /home/csgoserver/serverfiles/bin/libgcc_s.so.1: version `GCC_7.0.0' not found (required by /lib/i386-linux-gnu/libstdc++.so.6)
L 01/20/2022 - 14:31:29: Detected engine 11 but could not load: /home/csgoserver/serverfiles/bin/libgcc_s.so.1: version `GCC_7.0.0' not found (required by /lib/i386-linux-gnu/libstdc++.so.6)
L 01/20/2022 - 14:54:25: Detected engine 11 but could not load: /home/csgoserver/serverfiles/bin/libgcc_s.so.1: version `GCC_7.0.0' not found (required by /lib/i386-linux-gnu/libstdc++.so.6)
L 01/20/2022 - 16:31:05: Detected engine 11 but could not load: /home/csgoserver/serverfiles/bin/libgcc_s.so.1: version `GCC_7.0.0' not found (required by /lib/i386-linux-gnu/libstdc++.so.6)
L 01/20/2022 - 16:41:53: Detected engine 11 but could not load: /home/csgoserver/serverfiles/bin/libgcc_s.so.1: version `GCC_7.0.0' not found (required by /lib/i386-linux-gnu/libstdc++.so.6)
L 01/20/2022 - 17:13:28: Detected engine 11 but could not load: /home/csgoserver/serverfiles/bin/libgcc_s.so.1: version `GCC_7.0.0' not found (required by /lib/i386-linux-gnu/libstdc++.so.6)
L 01/20/2022 - 17:26:25: Detected engine 11 but could not load: /home/csgoserver/serverfiles/bin/libgcc_s.so.1: version `GCC_7.0.0' not found (required by /lib/i386-linux-gnu/libstdc++.so.6)
L 01/20/2022 - 17:43:29: Detected engine 11 but could not load: /home/csgoserver/serverfiles/bin/libgcc_s.so.1: version `GCC_7.0.0' not found (required by /lib/i386-linux-gnu/libstdc++.so.6)
L 01/20/2022 - 18:07:27: Detected engine 11 but could not load: /home/csgoserver/serverfiles/bin/libgcc_s.so.1: version `GCC_7.0.0' not found (required by /lib/i386-linux-gnu/libstdc++.so.6)
L 01/20/2022 - 18:09:46: Detected engine 11 but could not load: /home/csgoserver/serverfiles/bin/libgcc_s.so.1: version `GCC_7.0.0' not found (required by /lib/i386-linux-gnu/libstdc++.so.6)

Doesn't work.

**Comment by Anon-Exploiter** [comment]
Hi, 

What version of ubuntu are you trying this on? I recently remade by server from scratch and everything went fine. 

The script is specific to Ubuntu 20.04 rn.

**Comment by SpawnTerror** [comment]
Pardon my mistake, I didn't use the script. I ran everything standalone on 21.04. Found your issue here and the solution, but it just doesn't work on 21.04. I'm stuck and will probably have to spin 20.x to get SM and MM to run.

**Comment by Anon-Exploiter** [comment]
@SpawnTerror yep, I remember when I was upgrading from Ubuntu 18.04 to 20.04, this^ issue occurred. Do let me know if you solve this somehow, I'll add a note for it in the README for 21.04 users.

**Comment by SloudPL** [comment]
I found the fix just delete one file works on ubuntu 22.04
Delete Serverfiles/bin/libgcc_s.so.1 and start the server and metamod + sourcemod should work

**Comment by ptrsuk** [comment]
> I found the fix just delete one file works on ubuntu 22.04 Delete Serverfiles/bin/libgcc_s.so.1 and start the server and metamod + sourcemod should work

My man! Thank you so much for this, I've been looking through logs trying to figure out the problem for a few hours now and this solved my problem.

**Comment by SloudPL** [comment]
> > I found the fix just delete one file works on ubuntu 22.04 Delete Serverfiles/bin/libgcc_s.so.1 and start the server and metamod + sourcemod should work
> 
> My man! Thank you so much for this, I've been looking through logs trying to figure out the problem for a few hours now and this solved my problem.

What's funny, I couldn't find how to fix it too and I just saw that this file is missing in linux why not delete this file from csgo server and somehow it happened that it worked.

**Comment by TheZLabs** [comment]
> 

I have been trying to fix this for days. You just solved my problem. Thank you very much!
